docs/maintainer/infrastructure.md

```diff
@@ -33,42 +160,75 @@ Smithy contains maintenance code for conda-forge, which is used by the `conda-sm
+Its job is to procure an automation loop where some maintenance tasks are added.
+Its main user is the core team.
 
-### Documentation
+#### admin-requests
 
-The documentation website lives in [conda-forge/conda-forge.github.io](https://github.com/conda-forge/conda-forge.github.io/), and is automatically deployed to our [online version](https://conda-forge.org/).
+- ⚙️ Deployed in [Github Actions via `conda-forge/admin-requests`](https://github.com/conda-forge/admin-requests)
+- 🤖 Uses [`@conda-forge-curator`](https://github.com/apps/conda-forge-curator)
+- 🔒 Has access to Github API, Anaconda.org (conda-forge and cf-staging), Circle, Travis, Azure
 
-The documentation is built with Docusaurus and the sources files are located in the [`docs/`](https://github.com/conda-forge/conda-forge.github.io/tree/main/docs) directory of the repository.
+This repository hosts workflows that mainly run when triggered by a user-initiated action.
+This is usually done via a PR that, once approved, is merged and triggers the requested action (mark a package as broken, archive a feedstock, etc).
 
-If you found any typo error, unclear explanations or new topics that can be covered, you can suggest changes to the documentation. For more details, please refer to [Improve the documentation](../user/contributing.md#improve-docs).
+It also does the job of creating new feedstocks for recipes that have been merged in [`conda-forge/staged-recipes`](#staged-recipes).
+The [`create_feedstocks` workflow](https://github.com/conda-forge/admin-requests/blob/main/.github/workflows/create_feedstocks.yml) runs every 10 minutes to create the new feedstock repositories on the `conda-forge` organization.
+The core logic is defined in the Python script [`.github/workflows/scripts/create_feedstocks.py`](https://github.com/conda-forge/staged-recipes/blob/main/.github/workflows/scripts/create_feedstocks.py).
 
-<a id="dev-admservice"></a>
+#### autotick-bot
 
-<a id="admin-web-services"></a>
+- ⚙️ Deployed in [`regro/cf-scripts`](https://github.com/regro/cf-scripts)
+- ⛓ Needs [`regro/cf-scripts`](https://github.com/regro/cf-scripts), [`regro/cf-graph-countyfair`](https://github.com/regro/cf-graph-countyfair), [`conda-forge/conda-forge-pinning-feedstock`](https://github.com/conda-forge/conda-forge-pinning-feedstock)
+- 🤖 Uses [`@regro-cf-autotick-bot`](https://github.com/regro-cf-autotick-bot)
+
+:::note
+The older repo [`regro/autotick-bot`](https://github.com/regro/autotick-bot) is no longer in use;
+the bot now runs directly in `regro/cf-scripts`.
+:::
+
+#### webservices
+
+- ⚙️ Deployed in Heroku Dyno (`conda-forge.herokuapp.com`)
+- ⛓ Needs [`conda-forge/conda-forge-webservices`](https://github.com/conda-forge/conda-forge-webservices)
+- 🤖 Uses [`@conda-forge-webservices`](https://github.com/apps/conda-forge-webservices), [`@conda-forge-admin`](https://github.com/conda-forge-admin), [`@conda-forge-linter`](https://github.com/conda-forge-linter)
```

Comment:
We don't use tokens from conda-forge-linter anymore as far as I know.

Reply:
We also have a webservices GitHub app that does most GitHub API related tasks now.